vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -173,6 +168,10 @@ def save_guest_devices_inventory(hardware, hashes)
         # Save the hardware to force an id if not found
         hardware.save! if hardware.id.nil?
         h[:network][:hardware_id] = hardware.id
+      else
+        # ensure save_network_inventory gets called
+        # it will be deleted
+        h[:network] = {}
```

Comment:
@Fryguy 
Note, the bug shows it self twice in [save_inventory.rb](https://github.com/ManageIQ/manageiq/blob/202b07ae5d89225be10470771a5a9da64ecf370c/vmdb/app/models/ems_refresh/save_inventory.rb#L161)

save_children no longer calls into children if the hash is nil. So we no longer clear out network inventory.